docs/reference/changelog.rst

```diff
@@ -70,23 +70,123 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
-8.6.0 (2025-Jan-20)
+8.6.0 (2025-Feb-05)
 -------------------
 
 Core
 ====
 
-* Improved error messaging when unable to pack.
+.. note::
+
+  Unless stated otherwise, the changes in this section apply to snaps using
+  ``core22`` or ``core24`` bases.
+
+* Add ``chisel`` as a ``build-snap`` if any part has slices and ``chisel``
+  isn't already listed as a ``build-snap``.
 
 Plugins
 #######
 
+Ant
+"""
+
+* Set ``JAVA_HOME`` environment variable to allow the
+  :ref:`Ant plugin <craft_parts_maven_plugin>` to detect the latest available
+  JDK.
+
+Go Use
+""""""
+
+* Add a new :ref:`Go Use plugin<craft_parts_go_use_plugin>` for setting up a
+  `workspace <https://go.dev/ref/mod#workspaces>`_ for Go modules.
+
+Maven
+"""""
+
+* Set ``JAVA_HOME`` environment variable to allow the
+  :ref:`Maven plugin <craft_parts_maven_plugin>` to detect the latest available
+  JDK.
+
+Rust
+""""
+
+* Fix dependency validation for the ``rust`` plugin when a ``rust-deps``
+  part exists (`canonical/craft-parts#890`_).
+
+Poetry
+""""""
+
+* Add new ``poetry-export-extra-args`` and ``poetry-pip-extra-args`` keys
+  (`canonical/craft-parts#932`_).
+
+* Fix an issue where the :ref:`poetry plugin <craft_parts_poetry_plugin>` would
+  try to install poetry from the package repositories when ``poetry-deps`` was
+  declared as a dependency (`canonical/charmcraft#1996`_).
+
 uv
 ""
 
-* Add a new plugin for projects using `uv <https://docs.astral.sh/uv/>`_.
-  For more information, see :doc:`uv plugin reference
-  </reference/plugins/uv_plugin>`.
+* Add a new :ref:`uv plugin<craft_parts_uv_plugin>` for projects using
+  `uv <https://docs.astral.sh/uv/>`_.
+
+Extensions
+##########
+
+KDE Neon 6
```

Comment:
Related to the extension migration: is this the preferred name? I went with "Qt 5 with KDE Neon [x]" but I'm quite happy to trim it.

Reply:
Ah, right.  Previously, the name was `kde-neon-6` and the title was informally `KDE Neon 6`.  I like your title because it's more descriptive.  My only concern is that it may mislead someone that they can use this extension for QT6 apps that aren't KDE applications.

@lengau - thoughts?

Reply:
I would be hesitant to headline this as something containing Qt since it does pull in the full KDE Neon content snap, especially since there's work on a separate Qt content snap.

Reply:
Leaving it as-is for now.